src/controllers/user.js

```diff
@@ -0,0 +1,76 @@
+import { User } from '../database/models';
+
+exports.getProfile = async (req, res) => {
+  try {
+    res.status(200).json({
+      status: 'success',
+      profile: req.user,
+    });
+  } catch (error) {
+    res.status(400).json({
+      status: 'success',
+      message: error.message,
+    });
```

Comment:
how can we have status of success yet we are catching error, use message when display success result otherwise use error